app.py

```
@@ -1,23 +1,71 @@
+            if os.path.exists(old_file):
+                os.remove(old_file)
+        
+        user.profile_image = filename
+        db.session.commit()
+        
+        return jsonify({
+            'message': 'Profile photo updated successfully',
+            'photo_url': f"/uploads/{filename}"
+        })
+        
+    except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)